### ISSUE TITLE
this module produces npm warnings downstream.

### ISSUE THREAD

**Issue by cekvenich** [acornjs/acorn-import-meta#3]
Eg, Javascript-obfuscator uses acorn v7+, and you in here specify ^6.
This gives everyone downstream this warning:
`
npm WARN acorn-import-meta@1.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.`

Please update package json to support current versions of acorn.

**Comment by sanex3339** [comment]
It will be nice

**Comment by sanex3339** [comment]
Any news?

**Comment by sanex3339** [comment]
@adrianheine can you update it please?

**Comment by mondeja** [comment]
With Yarn:
```bash
warning "webpack-obfuscator > javascript-obfuscator > acorn-import-meta@1.0.0" has incorrect peer dependency "acorn@^6.0.0"
```

Ping @marijnh

**Comment by marijnh** [comment]
I don't have release rights with this package, you'll have to wait for @adrianheine

**Comment by adrianheine** [comment]
I published 1.1.0 fixing this.

**Comment by sanex3339** [comment]
Thank you!
